test/unit/common/Reducers.test.js

```diff
@@ -819,4 +819,12 @@ describe("Reducers", () => {
       assert.propertyVal(nextState, "hide", false);
     });
   });
+  describe("ASRouter", () => {
+    it("should listen for pref changes on AS_ROUTER_PREF_CHANGED", async () => {
+      const action = {data: {foo: "bar"}, type: at.AS_ROUTER_PREF_CHANGED};
+      const result = ASRouter({}, action);
```

Comment:
Ideally, there would be a newline between the `action` and `result` lines, to follow the semi-convention of splitting tests into

* setup
* execute
* verify

chunks